numpy/linalg/linalg.py

```diff
@@ -700,6 +709,14 @@ def cholesky(a):
        If the decomposition fails, for example, if `a` is not
        positive-definite.
 
+    See Also
+    --------
+    scipy.linalg.cholesky : Compute the Cholesky decomposition of a matrix.
+    scipy.linalg.cholesky_banded : Cholesky decompose a banded Hermitian
+        positive-definite matrix
+    scipy.linalg.cho_factor : Compute the Cholesky decomposition of a matrix,
+        to use in cho_solve
+
```

Comment:
In other places you align the second line to the `:`. Here you do not. Just wondering if we have a standard? Not worth another commit, but I always wonder how to indent a continuation line.

Reply:
I'll fix this.  These are the first lines that I added in the PR.  Later I noticed alignment on the colon used in other docstrings, so I started following that precedent, but I forgot to go back and fix this one.

Reply:
Fixed in the latest update.